site/src/testHelpers/entities.ts

```diff
@@ -395,13 +395,14 @@ export const MockWorkspace: TypesGen.Workspace = {
 
 export const MockStoppedWorkspace: TypesGen.Workspace = {
   ...MockWorkspace,
-  latest_build: MockWorkspaceBuildStop,
+  latest_build: { ...MockWorkspaceBuildStop, status: "stopped" },
 }
 export const MockStoppingWorkspace: TypesGen.Workspace = {
   ...MockWorkspace,
   latest_build: {
     ...MockWorkspaceBuildStop,
     job: MockRunningProvisionerJob,
+    status: "stopping",
```

Comment:
ideally we're pulling these values from the types that way they stay in sync

Reply:
Yeah, since the generated type is a union of strings, I don't know how to do that. Do you have a way? Or would I just have to create an enum to layer on top of the generated type?

Reply:
Hmm...we might have to dig in. At first glance, I'm not sure. I mean you do have the `TypesGen.Workspace` there so maybe we don't have to worry. We could look later!